modules/ols-creating-lightspeed-custom-resource-file-using-cli.adoc

```diff
@@ -60,6 +60,8 @@ spec:
   ols:
     defaultProvider: rhelai
     defaultModel: models/granite-7b-redhat-lab
+    additionalCAConfigMapRef:
+      name: openshift-service-ca.crt
```

Comment:
i think this is supposed to be the name of the configmap (in what namespace) and not a filename

Reply:
Thanks. We ended up breaking this out as a separate task.

This is done.